dev/lint-r.R

```diff
@@ -15,15 +15,18 @@
 # limitations under the License.
 #
 
+argv <- commandArgs(TRUE)
+SPARK_ROOT_DIR <- as.character(argv[1])
+
 # Installs lintr from Github.
 # NOTE: The CRAN's version is too old to adapt to our rules.
 if ("lintr" %in% row.names(installed.packages())  == FALSE) {
   devtools::install_github("jimhester/lintr")
 }
-library(lintr)
-
-argv <- commandArgs(TRUE)
-SPARK_ROOT_DIR <- as.character(argv[1])
 
+# NOTE: You should install SparkR with `R/install-dev.sh` before running this script.
+library(lintr)
+library(testthat)
+library(SparkR, lib.loc = file.path(SPARK_ROOT_DIR, "R", "lib"))
```

Comment:
Minor thing -- it would be good for the script to exit if any of these package loads fail. We can pass in `logical.return=T` to the `library` call to do this.